
try:
    print(num)
except Exception as result:
    print(result)
# 输出 name 'num' is not defined